contrib/download-frozen-image.sh

```diff
@@ -42,6 +42,8 @@ while [ $# -gt 0 ]; do
 	[ "$tag" != "$imageTag" ] || tag='latest'
 	tag="${tag%@*}"
 
+	imageFile="${image////_}" # "/" can't be in filenames :)
```

Comment:
Jeez, even Debian Squeeze (now `oldoldstable`) and Ubuntu Lucid (10.04) have Bash 4.0. :jack_o_lantern: